.github/workflows/create-versioning-pr.yaml

```diff
@@ -46,8 +46,8 @@ jobs:
       run: |
         version=$(jq -r .version ./charts/kubernetes-tentacle/package.json)
         version="$version" yq -i '.version = strenv(version)' ./charts/kubernetes-tentacle/Chart.yaml
-        git config user.name github-actions
-        git config user.email github-actions@github.com
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config user.name "github-actions[bot]"
```

Comment:
It is a standard email apparently 🤷 
https://github.com/orgs/community/discussions/26560#discussioncomment-3531273